microverse project

santitoot opened the pull request:
it's just a simple HTML and CSS project

santitoot committed:
change the name and delete lines -live demo-

santitoot committed:
Update README.md

santitoot committed:
the file was configured to have my information

santitoot committed:
fixing merge

Qoosim commented on README.md:
```diff
@@ -241,48 +149,31 @@ Feel free to check the [issues page](../../issues/).
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
- [ ] Kindly set up MIT License for your project by following this [guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). The link currently is broken.

![license](https://user-images.githubusercontent.com/26597432/234706982-cc4b33cd-c3cc-4d25-aa0a-22d1fd685bb0.png)

Qoosim reviewed (changes requested):
#  *Status: Required Changes* :recycle:

Hi @santitoot, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct . :heavy_check_mark: 

![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)


## Required Changes ♻️
_Check the comments under the review._

- [ ] It is required to give a class name to the `h1` tag in your index.html and use the class name to style the content of the `h1` tag.

![css-file](https://user-images.githubusercontent.com/26597432/234710315-09b4826b-c9f9-4c8c-97a0-33e612cab62b.png)

- [ ] It is also required that there should be linters files set up in your project. There is no linters files at this moment. Kindly follow this [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) to set it up properly.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

santitoot committed:
give a class name to the h1 tag in index.html and use the class name to style the content of the h1 tag

santitoot committed:
Create MIT.md

Created MIT License file as per the request of the code review

santitoot committed:
Update README.md

Updated Readme File to correct broken links in the license section as per the code review

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @santitoot, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif" width="600px" height="300px">

### To Highlight :1st_place_medal: 

- Descriptive proper Pull Request summary. ✔️  
- Hello Microverse project done ✔️ 
- There is a `.gitignore` file listing `test.md` file and `node_modules` ✔️
- Good use of Github flow ✔️ 

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Gambit142 reviewed (commented):
Hi @NewtonOmbese 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- It has HTML and CSS files ✔️
- It has node_module and test.md git ignored ✔️
- README.md file is professional ✔️ 
- Use of GitHub flow ✔️ 

### Optional Suggestion
- _**Kindly consider the comment tagged [OPTIONAL] under this review summary**_

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
